package.json

```diff
@@ -36,26 +36,32 @@
   "dependencies": {
     "@babel/runtime": "^7.13.9",
     "@esbuild-plugins/node-resolve": "0.0.14",
+    "@fal-works/esbuild-plugin-global-externals": "^2.1.1",
     "@mdx-js/mdx": "^1.6.22",
     "@mdx-js/react": "^1.6.22",
-    "esbuild": "^0.8.55",
+    "esbuild": "^0.8.57",
     "gray-matter": "^4.0.2",
-    "rollup": "^2.39.0",
-    "rollup-plugin-terser": "^7.0.2"
+    "xdm": "^1.5.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.8",
+    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
+    "@babel/plugin-transform-runtime": "^7.13.9",
```

Comment:
I'm not sure I know why these need to be added here.

Reply:
these are left over from me throwing things at the wall to see what sticks with _xdm_. I'll clear them out before the final submission.